### PR TITLE
Daily Summary Report

### PR DESCRIPTION
**Pull request by bradymiller**
Revision for the Daily Summary Report by rishabhemr

### CODE REVIEW

**Comment by bradymiller** [comment]
the prior revision and comments of the PR can be found here:
https://github.com/openemr/openemr/pull/169

File: library/appointments.inc.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
     'trackerstatus' => array( 'trackerstatus', 'date', 'time', 'patient' ),    
 );
 
-function fetchEvents( $from_date, $to_date, $where_param = null, $orderby_param = null, $tracker_board = false, $nextX = 0, $bind_param = null, $query_param = null )
+function fetchEvents( $from_date, $to_date, $where_param = null, $orderby_param = null, $tracker_board = false, $nextX = 0, $bind_param = null, $query_param = null,$facility_name = 0 )
```
Comment (bradymiller) [inline library/appointments.inc.php]: TODO for me:
Look into using fetchAppointments function instead and looking into the facility stuff to ensure it is optimized and working.

Comment (bradymiller) [inline library/appointments.inc.php]: TODO for me:
Also, if need the $facility_name, then will change it to a boolean.

Comment (bradymiller) [inline library/appointments.inc.php]: TODO for me:
Also add the menu item to the layout tab menu.

**Comment by bradymiller** [comment]
Hi,

I did some integration work and committed this to the codebase:
https://github.com/bradymiller/openemr/commit/599e54936692a888bd82d212c481b1a61c8b6cea

Note I removed the reliance on the facility boolean, and it tested well on both the Daily Summary and Appointments reports. Just let me know if you see issues:
https://github.com/bradymiller/openemr/commit/599e54936692a888bd82d212c481b1a61c8b6cea#diff-83fc2736799cf98b8087359e5f6a36b5

thanks for the very nice contribution!

-brady

**Comment by bradymiller** [comment]
@rishabhemr,
Just making sure you got saw the above posts.

**Comment by rishabhemr** [comment]
Hi Brady,

Thanks for your great supports & providing extensive code review comments to make the daily summary report successful.

Yes, we saw your above posts today. 

Please let us know if any further changes are needed in the daily summary reports.

-Zaver Vaghasiya

**Comment by rishabhemr** [comment]
Hi Brady & Team,

Thanks for all your efforts for giving us code review feedback. we hope the new code delivered is acceptable to you. Could you please let us know if anything else needs to be done from our side. We are happy to provide our support. 

Thanks
Rishabh Software

**Pull request closed without merging**